CHANGELOG.md

```diff
@@ -1,17 +1,29 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [5.0.2]
+
+### Fixed
+
+- fix: allow category extraction when scope is used ([#240](https://github.com/MetaMask/auto-changelog/pull/240))
```

Comment:
In my opinion this should be a feature/change, and the release should be a minor bump. What do you think?

Reply:
Ah sorry. I thought the PR was implementing this feature, I wasn't aware of that option.